OpenSourcePolitics/decidim-app
# Fix/import proposals in budget

#### :tophat: Description
Fix import of proposals in budgets: when selecting a "global" scope, proposals that belongs to this scope and proposals that belongs to a child scope are imported.

#### :pushpin: Related Issues
*Link your PR to an issue*
- [Notion card](https://www.notion.so/opensourcepolitics/bd4bf860eab94f9daca7da803f09e3b2?v=c96df65755dd403d95e86f82d7749709&p=80b2268cbc6842a2a123a15a75d0b409&pm=c)

#### Testing
1. Go to decidim-app as an admin
2. In the admin dashboard, go to Processes, and click on one process
3. See in the information page of the process that scopes are enabled and a global scope like "Florida" is selected
4. Go to the components page (= Fonctionnalités) and see that the scope "Florida" is also selected for every component created 
5. Go to a Budget component, click on Manage projects and click on Import proposals to budgets
6. Select the "Florida" scope and  import proposals into the budget 
7. See that accepted proposals with "Florida" scope are imported + accepted proposals with children scopes like "Remaside" are also imported


#### Tasks
